consensus/istanbul/validator/weighted.go

```diff
@@ -592,7 +641,7 @@ func (valSet *weightedCouncil) Refresh(hash common.Hash, blockNum uint64) error
 		return errors.New("skip refreshing proposers due to no staking info")
 	}
 
-	weightedValidators, stakingAmounts, err := valSet.getStakingAmountsOfValidators(newStakingInfo)
+	weightedValidators, stakingAmounts, err := getStakingAmountsOfValidators(valSet.validators, newStakingInfo)
```

Comment:
Does this `valSet.validators` always return the filtered validators?

Reply:
Yes, it is filtered validators by minimum staking.